news/2022-05-21-project-loved-may-2022.md

```diff
@@ -0,0 +1,321 @@
+243 BPM, 5:58 | 8.28★, 8.71★\
+*description written by [Kasumi-sama](https://osu.ppy.sh/users/6177263)*
+
+Resident osu!catch mapper Minato Yukina makes once again his return to the Loved section, with potentially his 20th Loved beatmap, *Contrapasso -purgatorio-*, which was mapped for [AxS' 4th edition](https://osu.ppy.sh/community/forums/topics/822420)'s Grand Finals tiebreaker. This map takes any osu!catch player to one hellish ride, with the first two kiai sections riddled with relatively hard antiflow patterns, jumps and bursts, and the map does not stop the intensity it pulls out in the slightest, even up to the very end, with the nastiest burst right at the end, ready to kill players.
+
+[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1107782.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Chihiro Nakamura - Kasaneteku](https://osu.ppy.sh/beatmapsets/1107782#fruits) by [Unlucky_w](https://osu.ppy.sh/users/4820793)\
+147 BPM, 3:43 | 6.71★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+This month we have a new face in the Loved category: Unlucky_w, formerly known as FastYoshi. And oh boy, this map is really interesting.
+
+First of all, this is perhaps one of the catchiest songs ever, and for that reason a good selling point for restarting the map over and over again until you can't get it out of your head. But not only that; the map itself is also catchy in a sense. It flows really well and fits the song nicely, highlighting the key parts of it. It also plays differently from most of the other maps of this day and age but in a really good way. It is not an easy map, but a very doable one with some practice.
+
+Truly a hidden gem, you should definitely give it a try (and get addicted to the song along the way).
+
+[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/945848.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Masayoshi Minoshima - Bad Apple!! feat. nomico (Camellia's "Bad Psy!!" Remix)](https://osu.ppy.sh/beatmapsets/945848#fruits) by [Kukkai](https://osu.ppy.sh/users/7811952)\
+160 BPM, 4:25 | 7.08★\
+*description written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+Right, let's keep this one short. You know Kukkai by now. No, actually, you love Kukkai by now. We know you do. So here's another one of their maps! *Bad Apple* is one of the most popular songs in the osu! community, so a Camellia remix (of all things) of it will never be received in a bad manner by you lovely people. This map is pure quality, despite the fact that it has some quirky streaming patterns in combination with some of the drops in the song. But hey, that's what Loved is for, right?! We are sure you'll like it. The only thing to note is that this is a relatively difficult map (though not extreme), so only fairly advanced players allowed for this one!
```

Comment:
```suggestion
Right, let's keep this one short. You know Kukkai by now. No, actually, you love Kukkai by now. We know you do. So here's another one of their maps! *Bad Apple* is one of the most popular songs in the osu! community, so a Camellia remix (of all things) of it will never be received in a bad manner by you lovely people.

This map is pure quality, despite the fact that it has some quirky streaming patterns in combination with some of the drops in the song. But hey, that's what Loved is for, right?! We are sure you'll like it. The only thing to note is that this is a relatively difficult map (though not extreme), so only fairly advanced players allowed for this one!
```